Makefile

```diff
@@ -14,7 +14,9 @@ thirdparty/%:
 	make -f Makefile.thirdparty $(@:thirdparty/%=%)
 
 clean:
-	rm -Rf build build-static debug-build
+	rm -Rf build build-static debug-build xbuild
```

Comment:
I must reject this change.
The target `clean` should be responsible for cleaning files created for building the program.
`xbuild/` is not created as a result of any make target in this Makefile.
Likely, files in `thirdparty/` are not built with the default target, or any other target except `thirdparty`, which is simply a shorthand to call make target in another Makefile`Makefile.thirdparty`.
In very rare cases (the Linux distro's software manager doesn't provide those third party packages, or has to use different build options) you'll need to build them by explicitly running `make thirdparty`; then there is a `make thirdparty/clean` for cleanup.